target/linux/ipq807x/files/arch/arm64/boot/dts/qcom/ipq8074-nbg7815.dts

```diff
@@ -0,0 +1,449 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/* Copyright (c) 2022, Karol Przybylski <itor@o2.pl> */
```

Comment:
Since there will be changes, bump to 2023

Reply:
Good catch, also updates the person list!